packages/atdpy/atdpy.2.5.0/opam

```diff
@@ -62,7 +62,7 @@ bug-reports: "https://github.com/ahrefs/atd/issues"
 depends: [
   "dune" {>= "2.8"}
   "ocaml" {>= "4.08"}
-  "atd" {>= "2.3.0"}
+  "atd" {= version}
```

Comment:
You probably already know this: to achieve this in the `dune-project` file, you can use the `(= :version)` constraint

Reply:
Thanks! I just discovered it after a failed attempt.